<launch>
	<arg name="world_name" default="$(find searcher)/worlds/dart_world.world"/>	
	<arg name="mobile_base" default="true" />
	<arg name="use_gpu_ray" default="false" />
  	<arg name="use_kinect" default="true" />
	
	<include file="$(find velmobil_navigation)/launch/laserscan_multi_merge.launch"/>
	<!-- launch omnivelma simulation -->
	<include file="$(find velma_common)/launch/velma_system.launch">
		<arg name="world_name" value="$(arg world_name)" />
		<arg name="mobile_base" value="$(arg mobile_base)" />
		<arg name="use_gpu_ray" value="$(arg use_gpu_ray)" />
    		<arg name="use_kinect" value="$(arg use_kinect)" />
		<arg name="gui" value="true"/>
		<arg name="GAZEBO_MODEL_PATH" value="$(find searcher)/models/models"/>
  		<arg name="GAZEBO_RESOURCE_PATH" value="$(find searcher))/models"/>
	</include>
		
				<!--UNIT BOX -->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_clone::link"/>
		<arg name="frame_id" value="table"/>
	</include>

				<!--UNIT BOX0-->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table_clone_clone::link"/>
		<arg name="frame_id" value="table2"/>
	</include>
				
				<!--JAR -->
   	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="jar::link"/>
		<arg name="frame_id" value="beer"/>
	</include>

				<!--RIGHT HANDLE CABINET -->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cabinet_door_fragile::right_handle"/>
		<arg name="frame_id" value="right_handle"/>
	</include>	

				<!--CABINET -->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cabinet_door_fragile::cabinet"/>
		<arg name="frame_id" value="cabinet"/>
	</include>	

				<!--RIGHT DOOR CABINET -->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cabinet_door_fragile::right_door"/>
		<arg name="frame_id" value="right_door"/>
	</include>	
	
				<!--CAFE TABLE -->
	<include file="$(find rcprg_gazebo_utils)/launch/gazebo_publish_ros_tf_object.launch">
		<arg name="link_name" value="cafe_table::link"/>
		<arg name="frame_id" value="cafe_table"/>
	</include>

</launch>
